packages/dapp/src/pages/Home.jsx

```diff
@@ -120,7 +159,7 @@ export const Home = () => {
             })}
         py="3rem"
         m="1rem"
-      ></Flex>
-    </Container>
+      ></Flex> */}
```

Comment:
Here's the last one. Not a big deal, I just prefer removing and going into history to reclaim old code.

Reply:
That's fair. I'll clean it up and remove the commented out sections. Forgot about them.